app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -556,50 +555,16 @@ public boolean isEditorHistoryEnabled() {
         return true;//getBool(R.string.pref_key__editor_history_enabled3, true);
     }
 
-    public int getEditorBasicColorSchemeId() {
-        return 0;
-    }
-
     public int getEditorForegroundColor() {
-        /*switch (getEditorBasicColorSchemeId()) {
-            default:
-            case 0:
-                return rcolor(darkMode ? R.color.white : R.color.dark_grey);
-            case 1:
-                return rcolor(darkMode ? R.color.white : R.color.black);
-            case 2:
-                return rcolor(R.color.solarized_fg);
-            case 3:
-                return rcolor(R.color.solarized_fg);
-        }*/
-
-
-        boolean darkMode = isDarkThemeEnabled();
-        int defval = rcolor(darkMode ? R.color.white : R.color.dark_grey);
-        return getInt(darkMode ? R.string.pref_key__editor_basic_color_scheme__fg_dark : R.string.pref_key__editor_basic_color_scheme__fg_light, defval);
+        return getInt(R.string.pref_key__editor_basic_color_scheme__fg, rcolor(R.color.primary_text));
```

Comment:
this looks like it reduces user configurable colors to only one (for either day/night, instead of one of day and one for night)